src/main/java/tasklist/TaskList.java

```diff
@@ -0,0 +1,142 @@
+package tasklist;
+import java.util.ArrayList;
+import java.util.List;
+
+import exception.DukeException;
+import task.Task;
+
+/**
+ * Represents a lists of Tasks
+ */
+public class TaskList {
+    private List<Task> tasks;
+
+    /**
+     * Instantiates a new TaskList with empty list
+     */
+    public TaskList() {
+        this.tasks = new ArrayList<Task>(100);
```

Comment:
I don't think there is a need to maintain the 100 size limit after changing to an 'ArrayList'